tests/acceptance/pageObjects/filesPage.js

```diff
@@ -153,6 +153,11 @@ module.exports = {
         .isVisible(selector, (result) => {
           callback(result.value)
         })
+    },
+    copyPermalink: function () {
```

Comment:
need a better name, as there are also links to the items in the side pannel

Reply:
renamed to `copyPermalinkFromFilesAppBar`